src/Raven.Server/Web/System/StudioHandler.cs

```diff
@@ -251,8 +254,6 @@ private void CacheCompress()
 
             HttpContext.Response.ContentType = GetContentType(Path.GetExtension(serverRelativeFileName));
             HttpContext.Response.Headers[Constants.Headers.Etag] = metadata.ETag;
-            // Disable chunked responses
-            HttpContext.Response.Headers[Constants.Headers.TransferEncoding] = "identity";
```

Comment:
This was added because Kestrel would chunk the response and screw the Gzip encoding if was not there (exactly WHY it would break it, I don't remember). When Kestrel chunked, both Firefox and Chrome would break.

Please make sure that this does not change anything in those browsers.

Reply:
It isn't chunked once you set content length. See: https://www.w3.org/Protocols/rfc2616/rfc2616-sec4.html#sec4.4
Messages MUST NOT include both a Content-Length header field and a non-identity transfer-coding. If the message does include a non- identity transfer-coding, the Content-Length MUST be ignored.